packages/ui-components/src/components/PageHeader/PageHeader.component.tsx

```diff
@@ -35,13 +34,21 @@ const logoContainerStyles = `
   [&>*]:jn-object-contain
 `
 
-const headingStyles = (clickable: boolean) => {
-  return `
-    jn-text-lg
-    jn-text-theme-high
-    ${clickable && "jn-cursor-pointer"}
-    `
-}
+const contentContainerStyles = `
+  jn-grid
+  jn-grid-cols-[1fr,minmax(0,max-content)]
```

Comment:
If no heading exists but there are children then the children will be dumped into the first grid column here because the div containing the heading is conditionally rendered (i.e. it doesn't exist at all if no heading was passed). 

I think we might be able to fix this by defining named grid areas and ensuring the children always get rendered in the second column. Or, probably easier, always render the heading div, even if it's empty.

Reply:
Absolutely! 

I have removed the conditional rendering logic for the `.juno-pageheader-heading` div, so it will always be there, preventing the options from slipping into the second grid column and thus preserving their position ont he right of the header at all times.